tern/analyze/common.py

```diff
@@ -268,6 +279,25 @@ def add_base_packages(image_layer, binary, shell):
                 listing_key=binary), 'error'))
 
 
+def get_package_files(layer_file_list, pkg_dict):
+    '''For each file in a package fetch its details from the
+    layer file list'''
+
+    files = pkg_dict['files']
+    new_layer_file_list = []
+
+    for f in files:
+        file_list = f.split('\n')
+        new_fd_list = []
+        for file in file_list:
+            for fd in layer_file_list:
+                if file == '/'+fd.path:
+                    new_fd_list.append(fd)
+        new_layer_file_list.append(new_fd_list)
+
+    pkg_dict['files'] = new_layer_file_list
```

Comment:
It looks like in this function, for each package a list of FileData objects are created and directly assigned to the `pkg_dict['files']` dictionary. In the spirit of making this more readable, maybe return `new_layer_file_list` or split this up into several other functions? Actually, if you implement `pkg.fill` in such a way that it also fills the file level information, then maybe you will not need this as `add_base_packages` will take care of that for you.